MarkdownEditor.elm

```diff
@@ -0,0 +1,39 @@
+module MarkdownEditor where
+-- Elm’s "import" keyword works mostly like "require" in node.js.
+-- The “exposing (..)” option specifies that we want to bring the Html module’s contents
+-- into this file’s current namespace, so that instead of writing out
+-- Html.form and Html.label we can just use "form" and "label" without the "Html." prefix.
+import Html exposing(..)
+-- This works the same way; we also want to import the entire
+-- Html.Events module into the current namespace.
+import Html.Events exposing (..)
+-- With this import we are only bringing a few specific functions into our
+-- namespace, specifically "id", "type'", "for", "value", and "class".
+import Html.Attributes exposing (id, type', for, value, class, style)
+-- startapp is boilerplate that most elm apps will use to follow the recommended
+-- elm architecture
+import StartApp.Simple as StartApp
+-- creates a view model that is a navbar
+view model =
+    div
+    [
+        navStyle
+    ][
+    ]
+
+navStyle: Attribute
+navStyle =
+    style
+    [
+        ("position",        "absolute")
```

Comment:
im not a fan of that proposed style in the elm docs... i think its easier to read when they all line up properly